src/FGame.cpp

```diff
@@ -75,21 +73,26 @@ void FGame::processEvents()
 	}
 	
 	// Check keyboard press
+	auto characters = m_engine->getCharacters();
 	if (sf::Keyboard::isKeyPressed(sf::Keyboard::Left))
 	{
-		//TODO
+		std::cout << "moving left";
+		characters[0].move(FDirection::left);
```

Comment:
There's also something wrong with moving. This call seems to register all the way to `FObject accelerate()` but velocity is never changed because of it for some reason.